_posts/2020-06-01-totw-5.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Did someone ask for TotW 5? (Feels like an odd / old thing, nothing objectionable but more historical than anything.)

That said, I could have cited it in a review yesterday and saved some time, so there's some value here I suppose, even if the examples are all C++11-quoting ...

Reply:
No, but it was referenced in 116 so I brought it along. And all else equal, I'd prefer to publish anything we still think is accurate